src/poprox_storage/repositories/newsletters.py

```diff
@@ -59,14 +59,17 @@ def fetch_newsletters(self, accounts: list[Account]) -> dict[UUID, dict[UUID, li
         for row in newsletter_result:
             raw_articles = []
             for article_json in row.content:
+                if isinstance(article_json, dict):
+                    article_json = json.dumps(article_json)
                 raw_articles.append(json.loads(article_json))
             articles = [
                 Article(
+                    article_id=raw["article_id"],
                     title=raw["title"],
-                    content=raw.get("description", None),
+                    content=raw.get("content", None),
```

Comment:
I'm not sure about this line—I know it looked weird but my recollection is that this was an artifact of different terminology between us and AP. What we call the `content` is really the blurb/slug and I don't think they call it that. If this is working the way you expect then it's probably okay, but just want to double-check this part.

Reply:
Ah gotcha! I'm applying this change since if we don't change it, the new line in the test would fail and return none:

`assert "article content 1" == user_1_newsletter[newsletter_1_id][0].content`

Is that expected or am I calling the wrong field in the test? I've printed out the Article object and looks like it just has `content` field instead of `description`.

Reply:
For the short term, I think that line should be:
```
assert "article content 1" == user_1_newsletter[newsletter_1_id][0].description
```

Outside the scope of this PR, we should probably clarify the names of the fields in the `Article` class to make this easier to figure out

Reply:
Hmm this is weird, since if I change it to `description`, we'll see such error:

`tests/test_newsletters.py::test_fetch_newsletters - AttributeError: 'Article' object has no attribute 'description'`

Referring to the [article object definition](https://github.com/CCRI-POPROX/poprox-concepts/blob/c0552428f046dcb1f706f421b3ae637e8022d9a6/src/poprox_concepts/domain/article.py#L28), indeed that's what this error message is complaining about....do I miss anything here?

Reply:
Maybe I'm misunderstanding which data this is operating on! I thought it was comparing Article objects to AP JSON, but I'm realizing now maybe it's comparing Article objects to serialized versions of the same objects?

Reply:
Yeah I think it should be the same object serialization case as you mentioned. So the current setup using `content` in all field to construct the `Article` object can pass, while the `description` would fail. If the current setup looks good I think we're ready to merge it!